astronomer/providers/apache/livy/operators/livy.py

```diff
@@ -0,0 +1,81 @@
+"""This module contains the Apache Livy operator async."""
+from typing import Any, Dict
+
+from airflow.exceptions import AirflowException
+from airflow.providers.apache.livy.operators.livy import LivyOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.apache.livy.triggers.livy import LivyTrigger
+
+
+class LivyOperatorAsync(LivyOperator):
+    """
+    This operator wraps the Apache Livy batch REST API, allowing to submit a Spark
+    application to the underlying cluster asynchronously.
+
+    :param file: path of the file containing the application to execute (required).
+    :param class_name: name of the application Java/Spark main class.
+    :param args: application command line arguments.
+    :param jars: jars to be used in this sessions.
+    :param py_files: python files to be used in this session.
+    :param files: files to be used in this session.
+    :param driver_memory: amount of memory to use for the driver process.
+    :param driver_cores: number of cores to use for the driver process.
+    :param executor_memory: amount of memory to use per executor process.
+    :param executor_cores: number of cores to use for each executor.
+    :param num_executors: number of executors to launch for this session.
+    :param archives: archives to be used in this session.
+    :param queue: name of the YARN queue to which the application is submitted.
+    :param name: name of this session.
+    :param conf: Spark configuration properties.
+    :param proxy_user: user to impersonate when running the job.
+    :param livy_conn_id: reference to a pre-defined Livy Connection.
+    :param polling_interval: time in seconds between polling for job completion. Don't poll for values >=0
```

Comment:
This need updating like we discussed this morning

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/57986db07290419021a0c3a2311a3daf88d2e488)